src/NHibernate/Loader/Hql/QueryLoader.cs

```diff
@@ -206,7 +205,7 @@ protected internal override ICollectionPersister[] CollectionPersisters
 
 		protected override IDictionary<string, string[]> GetCollectionUserProvidedAlias(int index)
```

Comment:
Shall this be made obsolete?

Reply:
Let's keep it as feature. It's still might be useful.